armi/reactor/blocks.py

```diff
@@ -601,6 +612,11 @@ def coords(self, rotationDegreesCCW=0.0):
         .. impl:: Coordinates of a block are queryable.
             :id: I_ARMI_BLOCK_POSI1
             :implements: R_ARMI_BLOCK_POSI
+
+            Calls to the :py:meth:`~armi.reactor.grids.locations.IndexLocation.getGlobalCoordinates`
+            method of the block's ``spatialLocator`` attribute, which recursively
+            calls itself on all parents of the block to get the coordinates of the
+            block's centroid in 3D cartesian space.
```

Comment:
We might want to add that rotated coordinates are not supported or something similar.

Reply:
https://github.com/terrapower/armi/pull/1619/commits/2dd57142bfd4ed8eac5a12395e45404732a2cc71